Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -7775,7 +7775,7 @@ TrustedTypesEnabled:
     WebKitLegacy:
       default: false
     WebKit:
-      default: true
+      default: WebKit::defaultTrustedTypesEnabled()
     WebCore:
       default: true
```

Comment:
I feel like we usually leave this `false` in WebCore in these cases. Hopefully `true` still inherits the same way?